src/piggui.rs

```diff
@@ -314,10 +309,25 @@ fn get_hardware_target(matches: &ArgMatches) -> HardwareTarget {
     target
 }
 
-#[cfg(not(target_arch = "wasm32"))]
 /// Parse the command line arguments using clap
 fn get_matches() -> ArgMatches {
-    let app = clap::Command::new(env!("CARGO_BIN_NAME")).version(env!("CARGO_PKG_VERSION"));
+    let app = clap::Command::new(env!("CARGO_BIN_NAME")).override_help(
```

Comment:
We don't need to override help, if we enable help with the feature changes, no?

Is this PR done and ready for final review & merge?